packages/@react-stately/virtualizer/src/OverscanManager.ts

```diff
@@ -50,42 +34,21 @@ export class OverscanManager {
     this.visibleRect = rect;
   }
 
-  collectMetrics() {
-    let time = performance.now() - this.startTime;
-    if (time < 500) {
-      this.averagePerf.addSample(time);
-    }
-
-    if (this.visibleRect.height > 0) {
-      let o = Math.abs(this.velocity.y * (this.averageTime.value + this.averagePerf.value));  
-      this.overscanY.addSample(o);
-    }
-
-    if (this.visibleRect.width > 0) {
-      let o = Math.abs(this.velocity.x * (this.averageTime.value + this.averagePerf.value));  
-      this.overscanX.addSample(o);
-    }
-  }
-
   getOverscannedRect() {
     let overscanned = this.visibleRect.copy();
 
-    let overscanY = Math.round(Math.min(this.visibleRect.height * 2, this.overscanY.value) / 100) * 100;
-    if (this.velocity.y > 0) {
-      overscanned.y -= overscanY * 0.2;
-      overscanned.height += overscanY + overscanY * 0.2;
-    } else {
+    let overscanY = this.visibleRect.height / 3;
+    overscanned.height += overscanY;
```

Comment:
~~Curious about the math here, is this essentially signalling to the virtualizer that the available rect is an additional 1/3 of the visible rect bounds in the direction of the scrolling~~ Just saw you additional comment. Was the 1/3 just a number that felt reasonable performance wise? Is there any downside to returning 0 for overscanY if the velocity is 0 (aka the user isn't scrolling)? I suppose its nice to always have the extra rows rendered at all times and to avoid changing this overscan value even more